packages/@react-spectrum/actiongroup/docs/ActionGroup.mdx

```diff
@@ -0,0 +1,196 @@
+
+```tsx example
+<ActionGroup>
+  <Item uniqueKey="brush" textValue="Brush"><Brush /></Item>
+  <Item uniqueKey="select" textValue="Select"><Select /></Item>
+  <Item uniqueKey="regionSelect" textValue="Select Region"><RegionSelect /></Item>
+</ActionGroup>
+```
+
+### Internationalization
+
+In order to internationalize an ActionGroup, the strings of all child items should be localized.
+
+## Events
+
+Use the `onSelectionChange` prop as a callback to handle press events on items.
+The `uniqueKey` prop from the selected item will be passed into the callback.
+
+The following example uses the `onSelectionChange` prop to update a string stored in React state.
+
+```tsx example
+function Example() {
+  let [lastKey, setLastKey] = React.useState('');
+  return (
+    <div>
+      <ActionGroup onSelectionChange={setLastKey}>
+        <Item uniqueKey="add">Add</Item>
+        <Item uniqueKey="delete">Delete</Item>
+        <Item uniqueKey="edit">Edit</Item>
+      </ActionGroup>
+      <p>Last selected: <span>{lastKey}</span></p>
+    </div>
+  );
+}
+```
+
+For multiple selections, use the `selectionMode` prop with the `multiple` value.
+```tsx example
+function Example() {
+  let [selectedKeys, setSelectedKeys] = React.useState([]);
+  return (
+    <div>
+      <ActionGroup selectionMode="multiple" onSelectionChange={(...args) => setSelectedKeys(args)}>
+        <Item uniqueKey="add">Add</Item>
+        <Item uniqueKey="delete">Delete</Item>
+        <Item uniqueKey="edit">Edit</Item>
+      </ActionGroup>
+      <p>Selected: {selectedKeys.map(k => <span key={k}>{k}</span>)}</p>
+    </div>
+  );
+}
+```
+
+## Props
+
+<PropTable component={docs.exports.ActionGroup} links={docs.links} />
+
+## Visual Options
+
+### Quiet
+
+```tsx example
+<ActionGroup isQuiet>
+  <Item uniqueKey="add">Add</Item>
+  <Item uniqueKey="delete">Delete</Item>
+  <Item uniqueKey="edit">Edit</Item>
+</ActionGroup>
+```
+
+### Emphasis
+
+The additional styling provided by the `isEmphasized` prop is only applied when items are selected.
+
+```tsx example
+<ActionGroup isEmphasized defaultSelectedKeys={['add']}>
+  <Item uniqueKey="add">Add</Item>
+  <Item uniqueKey="delete">Delete</Item>
+  <Item uniqueKey="edit">Edit</Item>
+</ActionGroup>
+```
+
+### Disabled
+
+To disable the entire ActionGroup, use the `isDisabled` prop on the root.
+
+```tsx example
+<ActionGroup isDisabled>
+  <Item uniqueKey="add">Add</Item>
+  <Item uniqueKey="delete">Delete</Item>
+  <Item uniqueKey="edit">Edit</Item>
+</ActionGroup>
+```
+
+To disable individual items, a list of `disabledKeys` can be provided.
+
+```tsx example
+<ActionGroup disabledKeys={['add', 'delete']}>
+  <Item uniqueKey="add">Add</Item>
+  <Item uniqueKey="delete">Delete</Item>
+  <Item uniqueKey="edit">Edit</Item>
+</ActionGroup>
+```
+
+### Orientation
+
+Using the `orientation` prop with value `vertical` changes the alignment of the items to follow the y-axis.
+
+```tsx example
+<ActionGroup orientation="vertical">
```

Comment:
Might want to add a set width to this example so that the buttons don't stretch to fill the entire example width.